### DISCUSSION TITLE
Ratpack HTTP client library instrumentation requires 1.6

### DISCUSSION THREAD

**Discussion by anuraaga** [open-telemetry/opentelemetry-java-instrumentation#3767] [Q&A]
Ratpack added interception points for HTTP client in 1.6 for tracing.

https://ratpack.io/manual/current/api/ratpack/http/client/HttpClientSpec.html#requestIntercept(ratpack.func.Action)

1.6 was released on 12/2018. Any thoughts on raising the javaagent floor to 1.6 as well (simple to muzzle I believe by using these new methods by leveraging the library instrumentation for HTTP client instead of Netty) or should we keep javaagent at 1.4 and separate the library out as 1.6?

**Comment by trask** [comment]
no objection from me on raising the min supported Ratpack version to 1.6
